### ISSUE TITLE
API List page - API Console - Township of Langley

### ISSUE THREAD

**Issue by BK01** [BCDevExchange/Our-Project-Docs#90]
#### _Story:_

As an API owner, once the new API store is in place (end of December 2015) I wish to see work initiated on a console for the Township of Langley's Cemetery API. This relates to a new API store that is compatible with the API vs. the previous API store which was not compatible.
#### _Success Criteria:_
- New API console for the Township of Langley displayed in the net new API store using Swagger 2.0.
#### _Deliverables:_
-  New Swagger 2.0 definition file
-  Published console to new API store

**Comment by BK01** [comment]
Swagger definition ready for deployment to API list page. Do we first need to have a Township of Langley profile page?

**Comment by lmullane** [comment]
@BK01 @DavidR8 Yes, we need the profile page.

**Comment by BK01** [comment]
(Blocked) On hold until profile page created for Township of Langley.

**Comment by BK01** [comment]
Propose posting Langley API swagger definition to the BCDevX GitHub repo. If needed in the future (as-is or to update) this can be referenced to build an API console. Approval from Langley required.
